kubernetes-client/src/main/java/io/fabric8/kubernetes/client/Config.java

```diff
@@ -485,7 +490,7 @@ private static boolean loadFromKubeconfig(Config config, String context, String
           }
 
           config.getErrorMessages().put(401, "Unauthorized! Token may have expired! Please log-in again.");
-          config.getErrorMessages().put(403, "Forbidden! User "+currentContext.getUser()+ " doesn't have permission.");
+          config.getErrorMessages().put(403, "Forbidden! User " + currentContext.getUser() + " doesn't have permission.");
```

Comment:
Minor nitpick, Can we get rid of this hardcoding and use `HttpUrlConnection` statuses here and at other places also? ;-)